src/main/java/duke/task/DeadlineTask.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+/**
+ * Inherit from Task.
+ * This is a DeadlineTask.
+ * Has extra time field for the deadline of this task.
+ */
+public class DeadlineTask extends Task {
+    private String time;
```

Comment:
Maybe naming this `deadline` would be more informative?

Reply:
Right, makes more sense. Thanks for pointing that out!